pcdsdevices/ipm.py

```diff
@@ -136,6 +137,51 @@ def remove(self, moved_cb=None, timeout=None, wait=False):
             return (rmstatus & self.diode.remove(moved_cb=moved_cb,
                                                  timeout=timeout, wait=wait))
 
+    def target_in(self, target_num, moved_cb=None, timeout=None, wait=False):
+        """
+        Moves the target to one of the target positions. There are 4 targets
+        with different thinkness and absorbtion/signal.
+        The targets move vertically. To drive them in, use presets:
+        ipm.target_in(x), where x = target number
+
+        Parameters
+        -----------
+        target: int
+            Number of which target to move in.
+            Must be one of the valid target states: 1-4 or out: 5
+            (TARGET1, TARGET2, TARGET3, TARGET4, OUT) respectively
+
+        moved_cb : callable, optional
+            Function to be run when the operation finishes. This callback
+            should not expect any arguments or keywords.
+
+        timeout : float, optional
+            Maximum time for the motion. If `None` is given, the default value
+            of this positioners is used.
+
+        wait : bool
+            If `True`, block until move is completed.
+
+        Returns
+        -------
+        status: MoveStatus
+        """
+        status = self.target.move(target_num, moved_cb=moved_cb,
+                                  timeout=timeout, wait=wait)
+
+        # Add our callback if one was given
+        if moved_cb is not None:
+            status.add_callback(moved_cb)
```

Comment:
This and some of the other things here may be redundant with `target.move`'s handling of the keyword arguments

Reply:
Ohh ok, I should not need to handle the `moved_cb` and `wait` here...